app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,10 @@
+/********************************************************
+* Autolab Specific CSS variables *
+********************************************************/
+$autolab-red: rgba(153, 0, 0, 0.9);
+$autolab-subtle-red: #c87b7e;
+$autolab-subtle-gray: #eee;
+$autolab-highlight-gray: #ccc;
+$autolab-black-text: #212121;
+$autolab-blue-text: #0882af;
```

Comment:
Nit: this style is used in a couple of places, could we replace the occurences with the variable?
- `instructor_gradebook.css:109`
- `annotations.css:515`
- `style.css.scss:1467`
- `gradesheet.css.scss:310`